dtest_config.py

```diff
@@ -11,6 +11,7 @@
 
 class DTestConfig:
     def __init__(self):
+        self.sstable_format = "bti"
```

Comment:
in https://github.com/datastax/cassandra-dtest/pull/78/commits/b3b67b935df6b2ada8e9d676afbd65c4132ebef7#diff-5c97a63c102c347022ec7bf217cb8f8df10168240ebb20917ec46f6c1fd248bbR14

see https://github.com/datastax/cassandra-dtest/pull/78/commits/0f0035efd5f79c5e827aba8b7016a80534f2821f#diff-5c97a63c102c347022ec7bf217cb8f8df10168240ebb20917ec46f6c1fd248bbR113 and https://github.com/datastax/cassandra-dtest/pull/78/commits/0f0035efd5f79c5e827aba8b7016a80534f2821f#diff-5c97a63c102c347022ec7bf217cb8f8df10168240ebb20917ec46f6c1fd248bbR113